### PR TITLE
Add button link on qwant logo

### PR DESCRIPTION
**Pull request by G-Ray**
## Description
Transform logo into a button which navigate to the homepage.

Please note in the animation below, there is no extra padding around the logo. Is it ok ?

## Why
UX.

## Screenshots
Peek 2020-12-02 18-05

### CODE REVIEW

File: views/index.ejs [hunk 1]
```diff
@@ -38,11 +38,15 @@
     <div id="react_root"></div>
     <div class="react_modal__container"></div>
     <div class="top_bar">
-      <form onsubmit="return false" novalidate class="search_form"
-        <% if(config.app.versionFlag) { %>
-          data-flag-text="<%= config.app.versionFlag %>"
-        <% } %> >
-        <div class="search_form__logo"></div>
+      <form onsubmit="return false" novalidate class="search_form">
+        <button
```
Comment (bbecquet) [inline views/index.ejs:42]: Let's add a title to it like we did on other buttons. Especially as there is no inner content.
What about `title="<%= _('Qwant Maps home') %>"`? Or `'Back to home'` ?

Comment (G-Ray) [reply views/index.ejs:42]: wdyt about just "Qwant Maps" ?

Comment (bbecquet) [reply views/index.ejs:42]: The goal is to describe the action. "Qwant Maps" seems quite ambiguous. But we can ask the design team.

Comment (G-Ray) [reply views/index.ejs:42]: I asked the design team. For now, we decided to go with your proposition `Qwant Maps Home`

**Review by bbecquet — APPROVED** [review]

**Pull request merged**